interfaces/css-typed-om.idl

```diff
@@ -165,9 +165,6 @@ enum CSSMathOperator {
     "max",
 };
 
-// FIXME: Uncomment this when IDLHarness supports CSS namespaces:
```

Comment:
This change blows up the test, there will be a single subtest failing with 'promise_test: Unhandled rejection with value: "CSS: namespace not yet supported"'. Blocked on https://github.com/w3c/web-platform-tests/issues/7583 in other words, although we can just back out this change and take the others.

Reply:
Let's unblock via manual curation, then, and look at the blocker later.